rohmu/atomic_opener.py

```diff
@@ -53,7 +37,7 @@ def atomic_opener(
 
 
 @overload
-def atomic_opener(
+def atomic_opener(  # type: ignore [misc]
```

Comment:
q: Why do we have to ignore that, what was the error?

Reply:
@akudiyar I believe this is something I had added to make mypy work with 3.7 setting as suggested by Roman, but since in the end I decided to just keep 3.9 and let a future PR decide which version we should target I guess this could be removed together with a few other comments.